oar/core/notification_mgr.py

```diff
@@ -0,0 +1,76 @@
+import os
+import smtplib
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+from oar.core.config_store import ConfigStore
+from oar.core.exceptions import NotificationException
+from oar.core.worksheet_mgr import WorksheetManager, WorksheetException
+from oar.core.worksheet_mgr import TestReport
+from gspread import Worksheet
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+class NotificationManager:
+    """
+    NotifiationManager will be used to send email or send message to slack to notificate related people
+    """
+    str1 = ''
+    def __init__(self, cs: ConfigStore):
+        self._cs = cs
+
+    def get_mail_content(self):
+        wm = WorksheetManager(self._cs)
+        report = wm.get_test_report()   
+        sheet_url = report.get_url()
+        mail_content = 'Hello ART team,\n''The '+version+' Z- stream release test status will be tracked in the following document:\n' +sheet_url
+        return print(mail_content)
+    
+    def subject_content(self):
+        """
+        Get subject content of email about sending report to art team
+        """
+        version = self._cs.release
+        sub_content = version+' z-stream errata test status'
+        return print(sub_content)
+   
+    def get_app_passwd(self):
+        """
+        Get google account application password
+        """
+        return self._cs.get_google_app_passwd()
+        
+              
+    def send_gmail(self, mail_content, sender_addr, recv_addrs, subject_content):
```

Comment:
@wewang58 error handling is missed, i cannot find try/except block any exception raised from email lib. we should catch all the exceptions raised and raise our own exception instead.